sound/pci/hda/patch_hdmi.c

```diff
@@ -2654,7 +2654,7 @@ static void generic_acomp_pin_eld_notify(void *audio_ptr, int port, int dev_id)
 	/* skip notification during system suspend (but not in runtime PM);
 	 * the state will be updated at resume
 	 */
-	if (snd_power_get_state(codec->card) != SNDRV_CTL_POWER_D0)
+	if (snd_power_get_state(codec->card) >= SNDRV_CTL_POWER_D3)
```

Comment:
maybe add an explicit Fixes tag to make sure this goes into linux-stable?

Reply:
@plbossart I thought about it, but the related patch from Takashi is itself good and was merged already in 2015. I added a "BugLink", this should have the same impact. Or I'll dig up the patch to change the HDMI codec for SOF, that would be more appropriate for the fixes.

But ack, I sent a RFC version for comments while waiting for more testing reports.